README.md

```diff
@@ -15,7 +15,9 @@ ManageIQ Python API Client package [manageiq-api-client-python] (https://github.
 
 ### Getting Started
 
-Currently, the module supports adding an OpenShift containers provider to manageiq.
+###### manageiq_provider module
+
+The `manageiq_provider.py` module currently supports adding, updating and deleting an OpenShift containers provider to manageiq.
```

Comment:
I'd also s/manageiq_provider.py/manageiq_provider/, as user cares what the module is called in playbooks, not which file implements it.  </nitpick>